### ISSUE TITLE
[Feature]: Enhance Accessibility Testing within PR flow

### ISSUE THREAD

**Issue by chrisreddington** [CloudWithChris/hugo-creator#59]
### Provide a general summary of the suggestion

Accessibility is a core aim of the theme, making sure that content is accessible to all.

### Context

Accessibility is something that all of us are responsible for, and have a duty to work on. To ensure the theme continues to aim for high standards, this should be a part of the PR check.

### Possible Implementation

- Microsoft Accessibility Insights as part of PR flow
- Google LightHouse as part of PR Flow

**Comment by chrisreddington** [comment]
Blocked by https://github.com/microsoft/accessibility-insights-action/issues/629
